python/glow/wgr/linear_model/logistic_model.py

```diff
@@ -163,6 +163,9 @@ def reduce_block_matrix(self, blockdf: DataFrame, labeldf: pd.DataFrame,
         transform_key_pattern = ['sample_block', 'label']
 
         if response == 'linear':
+            if not covdf.empty:
+                print('Ignoring covariates for linear response')
```

Comment:
It might make more sense to emit a warning here. Also, we should explain why in the function documentation for the user facing method.